File: lc1662.java
public class lc1662 {
//    1662. Check If Two String Arrays are Equivalent
    public static void main(String[] args) {
        String word1[]={"abc", "d", "defg"};
        String word2[]={"abcddefg"};
        String a="";
        String b="";
        for (int i=0;i< word1.length;i++){
            a=a+word1[i];
        }
        for (int i=0;i< word2.length;i++){
            b=b+word2[i];
        }
        boolean ans=false;
        if (a.equals(b)){
            ans=true;
        }
        System.out.println(ans);
    }
}

File: lc1662.py
from typing import List


class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        a,b="",""
        for i in word1:
            a+=i
        for i in word2:
            b+=i
        return a==b
word1,word2= ["a", "cb"], ["ab", "c"]